config/locales/en.yml

```
@@ -67,11 +67,18 @@ en:
           institute:
             label: "Institute"
             placeholder: "Enter institute..."
-    project:
+      project:
       show_more: "Show More"
       options:
```

YAMLlint (1.37.1)

[error] duplication of key "options" in mapping

(key-duplicates)